modules/RTC/RTCUtils.js

```diff
@@ -1271,28 +1313,35 @@ class RTCUtils extends Listenable {
      * logic compared to use screenObtainer versus normal device capture logic
      * in RTCUtils#_newGetUserMediaWithConstraints.
      *
-     * @param {Object} desktopSharingExtensionExternalInstallation
-     * @param {string[]} desktopSharingSources
+     * @param {Object} options
+     * @param {Object} options.desktopSharingExtensionExternalInstallation
+     * @param {string[]} options.desktopSharingSources
+     * @param {Object} options.frameRate
+     * @param {Object} options.frameRate.min - Minimum fps
+     * @param {Object} options.frameRate.max - Maximum fps
```

Comment:
I think that's options.gumOptions.frameRate.{min,max}, no?

Reply:
fixed!